docs/Autotune.md

```diff
@@ -1,40 +1,41 @@
-# Autotune
+# Autotune (Rev. 2015-03-04)
 
 Autotune helps to automatically tune your multirotor.
 
 ## Configuration.
 
 Autotune only works in HORIZON or ANGLE mode, before using auto-tune it's best you setup so there is as little drift as possible.
-Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support 
-pid_controller 2 (pid_controller 0 is the Cleanflight default).
+Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support pid_controller 2  or higher (pid_controller 0 is the Cleanflight default, pid_controller 1 will work for autotune as well).
```

Comment:
I think it actually supports every controller but controller 2 (controller 2 isn't supported because it uses floating point PID settings, but no other controller uses those).